index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for @naytev/grapheme-splitter
```

Comment:
Thank you very much for the contribution!

A single nit, the npm package is now just "grapheme-splitter", it was moved several months ago. The @naytev one is outdated

I'm not familiar with TypeScript. Do you think the instructions in the README need to be updated as well?

Reply:
I've added some usage notes to the file, I don’t think that it needs any instructions adding to the readme, but maybe a note that the module includes it's own TS declaration file.